src/pages/api/iexec/fund-release.tsx

```diff
@@ -0,0 +1,121 @@
+import mongoose from 'mongoose';
+import { getNewPayment } from '../../../queries/payments';
+import { EmailType, IPayment, PaymentTypeEnum, Web3mailPreferences } from '../../../types';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { sendMailToAddresses } from '../../../scripts/iexec/sendMailToAddresses';
+import { getUserWeb3mailPreferences } from '../../../queries/users';
+import { calculateCronData } from '../../../modules/Web3mail/utils/cron-utils';
+import {
+  checkPaymentExistenceInDb,
+  persistCronProbe,
+  persistEmail,
+} from '../../../modules/Web3mail/utils/database-utils';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const chainId = process.env.NEXT_PUBLIC_NETWORK_ID;
```

Comment:
NEXT_PUBLIC_NETWORK_ID doesn't exist anymore, we got not NEXT_PUBLIC_DEFAULT_CHAIN_ID because starterkit handle multichain. So you can use this one